paper/paper.md

```diff
@@ -0,0 +1,105 @@
+    orcid: 0000-0002-4465-2284
+    affiliation: 1
+  - name: Kaspar Beelen
+    orcid: 0000-0001-7331-1174
+    affiliation: 3
+  - name: Kalle Westerling
+    orcid: 0000-0002-2014-332X
+    affiliation: 1
+  - name: Rosie Wood
+    orcid: 0000-0003-1623-1949
+    affiliation: 1
+  - name: Kasra Hosseini
+    orcid: 0000-0003-4396-6019
+    affiliation: "1, 4"
+affiliations:
+ - name: The Alan Turing Institute, UK
+   index: 1
+ - name: Lancaster University, UK
+   index: 2
+ - name: School of Advanced Study, University of London, UK
+   index: 3
+ - name: Zalando SE, Germany
+   index: 4
+date: 11 December 2023
+bibliography: paper.bib
+
+
+---
+
+# Summary
+
+MapReader is an interdisciplinary software library for processing digitized maps `[@Hosseini_mapreader]`, but also other types of images, by 'patching' them into small, custom-sized cells, which are then classified according to the user's needs. MapReader thus offers a flexible pipeline which can be used both for manual annotation of small datasets as well as for Computer Vision-based inference of large collections. As an example, in `@Hosseini_mapreader`, we utilized MapReader's interface to manually annotate 62,020 patches, used its functionalities to train a suite of computer vision models and performed model inference on approximately 30.5 million patches. 
+
+MapReader's approach was inspired by methods in biomedical imaging, which were adapted for use by historians, and it is suitable for a wide range of applications in image analysis: it has, for example, been applied to an image classification problem in plant phenotype research `[@Corcoran]`. This cross-pollination between the humanities and the natural sciences was made possible by the open and reproducible research methods at the heart of MapReader.
+
+MapReader pioneers a methodological shift in how historians interact with maps as primary sources. Sustained engagement with big collections of maps rarely moves beyond analysis of cartographic history. To change this, MapReader encourages historians to reflect on the content of maps and is designed to facilitate linking datasets representing visual map content with other historical geospatial data.
+
+In this paper, we present the MapReader release at the conclusion of the Living with Machines project, which supported the development of the software and associated research. This release represents the culmination of extensive work to improve MapReader's usability, especially through clear documentation and tutorials.
+
+![MapReader modules and input-outputs. Credit: Rosie Wood.\label{fig:modules}](https://hackmd.io/_uploads/HJWJatQEa.png)
+
+
+# Statement of need
+
+Since the 1990s, map libraries have been scanning maps and creating digital collections of these images. In 2023, there are more than a million images of maps in digital libraries and archives around the world, and yet it is very difficult for anyone to do more than browse them in a web viewer.
```

Comment:
@kmcdono2 maybe ref for `Since the 1990s, map libraries have been scanning maps and creating digital collections of these images.`?

Reply:
@kmcdono2 also a ref for `more than a million images` would be helpful

Reply:
Yes, we can add a ref for the first, but for the second this is based on my count. I'm writing about this elsewhere, but not yet published.